configure.ac

```diff
@@ -538,8 +538,8 @@ AC_ARG_ENABLE(thread-cs,
 	AC_HELP_STRING([--enable-thread-cs=type],
 			[Choose the method used for critical sections
                          and other atomic updates when multiple
-                         threads are present.  Values may be global
-                         (default), per-object, per-vci, lock-free]),,enable_thread_cs=global)
+                         threads are present.  Values may be default, global,
+                         per-object, per-vci, lock-free]),,enable_thread_cs=default)
```

Comment:
I think we should clarify that the "default" for CH3 and CH4 are different.

Reply:
Agreed. I'll update it.